packages/origin-ui-core/src/components/Organization/OrganizationView.tsx

```diff
@@ -70,23 +70,28 @@ export function OrganizationView() {
 
     useEffect(() => {
         const getOrganization = async () => {
-            const iRecOrg = await iRecClient.getRegistrations();
+            let iRecOrg;
```

Comment:
We should always add types to `let` variables. I thought that was enabled in the linter 🤔